spec/javascript/components/lease_ups/LeaseUpApplicationsPage.test.js

```diff
@@ -160,4 +160,49 @@ describe('LeaseUpApplicationsPage', () => {
     // if submit wasn't clicked we shouldn't trigger an API request
     expect(mockCreateFieldUpdateComment).not.toHaveBeenCalled()
   })
+
+  test('updates substatus and last updated date on status change', async () => {
+    // Get the status last updated date before we change it, to verify that it changed.
+    const dateBefore = wrapper.find(rowSelector).first().find('PrettyTime').text()
+
+    // Change status to Appealed and open up the modal
+    act(() => {
+      wrapper
+        .find(rowSelector)
+        .first()
+        .find('Select')
+        .instance()
+        .props.onChange({ value: 'Appealed' })
+    })
+    wrapper.update()
+    expect(wrapper.find(StatusModalWrapper).props().isOpen).toBeTruthy()
+
+    // Change the substatus
+    act(() => {
+      wrapper
+        .find('SubstatusDropdown')
```

Comment:
nit: can you change these to be the actual components instead of string selectors?